mini-frame.el

```diff
@@ -92,6 +92,10 @@ to determine background color of mini frame."
 Called if `mini-frame-show-parameters' doesn't specify background color."
   :type 'function)
 
+(defcustom mini-frame-border-color "gray50"
```

Comment:
Let's name it `mini-frame-internal-border-color`. Also please make in `nil` by default.

Reply:
done